docs/docs/stories.mdx

```diff
@@ -54,6 +67,15 @@ You can use a response from your domain as an action by listing it as one
 in a story. Similarly, you can indicate that a story should call a custom action by including
 the name of the custom action from the `actions` list in your domain.
 
+You can also put bot utterance directly in the stories.
+
+:::caution
+This feature is experimental.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
```

Comment:
I would also put this under 2 options so that users know that it's only the 2nd option that is experimental. Also rephrase the warning as i've done above on the other one